src/main/java/com/odysseusinc/arachne/portal/config/WebSecurityConfig.java

```diff
@@ -255,10 +260,19 @@ protected void doFilterInternal(HttpServletRequest request, HttpServletResponse
         }
     }
 
+    @Autowired
+    AuthenticationSuccessHandler authenticationSuccessHandler;
+
     @Override
     protected void configure(HttpSecurity http) throws Exception {
-
         ExpressionUrlAuthorizationConfigurer<HttpSecurity>.ExpressionInterceptUrlRegistry reg = http
+                .requestCache().disable()  // TODO Remove before PR
+                .oauth2Login(login -> login
```

Comment:
This will be controlled by the configuration file. If there are no providers, it won't flip on.
We have to move the configuration options, such as client id and secret to external configuration anyway.